packages/web-client/app/components/card-pay/account-display/index.hbs

```diff
@@ -0,0 +1,66 @@
+<div class="account-display" ...attributes>
+  {{#if @balance}}
+    {{#if @isComplete}}
+      {{svg-jar @icon class="account-display__icon" role="presentation"}}
+      <div>
+        <div class="account-display__title-lg" data-test-account-title>
+          {{format-token-amount @balance}} {{uppercase @symbol}}.CPXD
+        </div>
+      </div>
+    {{else}}
+      {{svg-jar @icon class="account-display__icon-sm" role="presentation"}}
+      <div>
+        <div class="account-display__title-sm" data-test-account-title>{{uppercase @symbol}}.CPXD</div>
+        <div class="account-display__text-sm" data-test-account-balance>
+          {{format-token-amount @balance}} {{uppercase @symbol}}.CPXD
+        </div>
+      </div>
+    {{/if}}
+  {{else}}
+    {{#if @isComplete}}
+      {{#if @inner}}
+        {{svg-jar @icon class="account-display__icon" role="presentation"}}
+        <div>
+          <div class="account-display__title-lg" data-test-account-title>{{@title}}</div>
+          <div class="account-display__text account-display__address-wrapped" data-test-account-address>
+            {{@address}}
+          </div>
+        </div>
+      {{else}}
+        {{svg-jar @icon class="account-display__icon-sm" role="presentation"}}
+        <div>
+          <div class="account-display__title-sm" data-test-account-title>{{@title}}</div>
+          <div class="account-display__text-sm account-display__address account-display__text--outer" data-test-account-address>
+            {{truncate-middle @address 7 4}}
```

Comment:
Are `7` and `4` the standard way we use `truncate-middle`? If so, perhaps we should make those the default arguments.

Reply:
It's actually 6. I don't know where 7 came from. Updated the helper to use default values if none are given